llvmcpy/_generator.py

```diff
@@ -481,7 +486,7 @@ def clean_include_file(in_path: Path) -> None:
     shutil.move(out_path, in_path)
 
 
-def get_libraries() -> List[Path]:
+def get_libraries(context: Context) -> List[Path]:
```

Comment:
Push the function taking `context` to become `Context` methods?

Reply:
`get_libraries` could work, however there are functions like `parse_headers` which are basically half the code in this file.
I've made changes to have the least amount of line changed, if you want I can move also those functions to be part of `Context`

Reply:
Make a different commit, but go ahead.

Reply:
Done